packages/react-components/src/components/Input/AutoComplete.tsx

```diff
@@ -0,0 +1,225 @@
+import * as React from 'react';
+
+import {
+  FloatingNode,
+  FloatingPortal,
+  Placement,
+  Strategy,
+} from '@floating-ui/react';
+
+import { IPickerListItem, PickerList } from '../Picker';
+import { DEFAULT_LIST_HEIGHT, MIN_LIST_HEIGHT } from '../Picker/constants';
+import { useFloatingPicker } from '../Picker/hooks/useFloatingPicker';
+import { usePickerItems } from '../Picker/hooks/usePickerItems';
```

Comment:
These hooks, while not built specifically for this, work perfectly fine for building a "headless" `PickerList` (one without the usual `PickerTrigger`)